python/cudf/cudf/core/dataframe.py

```diff
@@ -6567,6 +6567,10 @@ def keys(self):
         return self._data.to_pandas_index()
 
     def itertuples(self, index=True, name="Pandas"):
+        """
+        Iteration is unsupported, See
+        :ref:`iteration <pandas-comparison/iteration>`
```

Comment:
Use complete sentences with periods. First line of the docstring should be on its own because it shows up as a method summary. (Same applies below.)
```suggestion
        Iteration is unsupported.
        
        See :ref:`iteration <pandas-comparison/iteration>` for more
        information.
```